config/config.go

```diff
@@ -109,6 +109,17 @@ var (
 	DefaultMarathonSDConfig = MarathonSDConfig{
 		RefreshInterval: Duration(30 * time.Second),
 	}
+
+	// The default Kubernetes SD configuration
+	DefaultKubernetesSDConfig = KubernetesSDConfig{
+		NodeMetricsPort: 10255,
```

Comment:
`KubeletPort` would be clearer. `NodeMetricsPort` could be read as the node exporter, something to do with node.js and as anything else that could be considered a node.